gradle/testing/randomization.gradle

```diff
@@ -106,6 +106,31 @@ allprojects {
   }
 }
 
+/**
+ * Inject location sensitive props through a provider to benefit from cache
+ * when locally cloning repo from 2 different locations
+ */
+class RandomizationEnvironmentProvider implements CommandLineArgumentProvider {
+
+    @Internal
+    String commonDir
+
+    @Internal
+    String securityPolicyFile
```

Comment:
My assumption was that this file is fairly static
https://github.com/jprinet/lucene/blob/feature/gradle_enterprise/gradle/testing/randomization/policies/tests.policy
but you're right, caching it might have some dramatic consequences in case of modification